package.json

```diff
@@ -840,7 +881,7 @@
         "json-schema-to-typescript": "^8.2.0",
         "lolex": "^4.2.0",
         "marked": "^0.7.0",
-        "mocha": "^6.2.0",
+        "mocha": "^6.2.3",
```

Comment:
Does this bump give us anything of value?

Reply:
That change is from master: 0f7e4fe549edd